src/input.rs

```diff
@@ -78,6 +78,10 @@ pub struct Opts {
     #[structopt(long)]
     pub accessible: bool,
 
+    /// Turns off Nmap.
+    #[structopt(long)]
+    pub without_nmap: bool,
```

Comment:
What do you think of calling this `no_nmap` and change `ignore_config` to `no_config`?  Maybe it's time to create a pattern for these optout defaults. :) 

We should also add this option to the `Config` struct so people can add it to their config files if they want.

Reply:
Sounds like a good idea to me! :) I'll change it when I wake up :D